test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -63,7 +63,7 @@ public void SimpleEndToEndOnnxConversionTest()
             var transformedData = model.Transform(data);
 
             // Step 2: Convert ML.NET model to ONNX format and save it as a file.
-            var onnxModel = mlContext.Model.ConvertToOnnx(model, data);
+            var onnxModel = mlContext.Model.ConvertToOnnxProtobufModel(model, data);
```

Comment:
>ConvertToOnnxProtobufModel [](start = 44, length = 26)

Seems ConvertToOnnxProtobuf is better. Model doesn't look like a part of a format's name.